spec/factories/miq_request_workflow.rb

```diff
@@ -18,4 +18,8 @@
   end
 
   factory :miq_provision_virt_workflow, :class => "MiqProvisionVirtWorkflow", :parent => :miq_provision_workflow
+
+  factory :miq_provision_virt_workflow_vmware,
```

Comment:
@djberg96 Do you see needing these for other provider types as well?

Reply:
Some of the other providers do support it, but I don't think it's necessary right now. Maybe in a separate PR.

Reply:
Agreed, it does not have to be this PR.  It just seemed like a good place to ask the question.  Thanks.